programs/github

```diff
@@ -176,12 +184,12 @@ if action == subcommands.auth then
 	local user, token = args[1], args[2]
 	if not user then return printError("No user specified.") end
 
-	if flags.delete then
+	if flags.remove then
 		github.Auth.delete(user)
 		print(('Deleted github token for user %s'):format(user))
 	else
 		if not token then return printError("No token specified.") end
```

Comment:
Should be possible to run something like `github auth eric --set-default` without a token